tests/test_fmi.py

```diff
@@ -91,6 +104,35 @@ def get_result(self):
         
 
 class Test_FMUModelME1:
+
+    @testattr(stddist = True)
+    def test_unzipped_fmu_exception_invalid_dir(self):
+        """ Verify that we get an exception if unzipped FMU does not contain modelDescription.xml, which it should according to the FMI specification. """
+        _helper_unzipped_fmu_exception_invalid_dir(FMUModelME1)
+
+    def _test_unzipped_fmu_helper(self, fmu_loader):
```

Comment:
Can we rename this? It looks like it tests bouncingBall from an unzipped folder. So I'd rather have _run_bouncing_ball(self, fmi_class) or something similar.

Reply:
I agree, it was a poor name for the helper function, changed it here:
https://github.com/modelon-community/PyFMI/commit/33c4c1efaa1b4faa235c2ab25f9b96c91c9ba94d

I also didn't change fmu_loader to fmi_class since for some of the tests we send in the function 'load_fmu' and I was thinking it might be misleading(?)

Reply:
Looks good!